etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
```

Comment:
wait, why the tst suffix? That would suggest the files are `.tst` files, which they are not, no?